pkg/cmd/admin/policy/modify_roles.go

```diff
@@ -78,6 +81,8 @@ func NewCmdAddRoleToUser(name, fullName string, f *clientcmd.Factory, out io.Wri
 	}
 
 	cmd.Flags().StringVar(&options.RoleNamespace, "role-namespace", "", "namespace where the role is located: empty means a role defined in cluster policy")
+	cmd.Flags().Var(&saNamesLong, "service-account", "service account in the current namespace to use as a user")
```

Comment:
I would like a shorter long name than --service-account, so I'm ok with you having a long and short one - maybe just do a single letter that is arbitrary like -c or something.